### PR TITLE
Update configuring-os2loop.md

### PR DESCRIPTION
**Pull request by Anna-itk**
Added description of differences between taxanomy terms

### CODE REVIEW

File: docs/configuring-os2loop.md [hunk 1]
```diff
@@ -24,10 +24,20 @@ In order to make the media library work, go to
 media categories for the site.
 
 Go to `/admin/structure/taxonomy` and make sure that the Profession and Subject
-taxonomies containt the terms needed for the content. Users can dynamically
+taxonomies contain the terms needed for the content. Users can dynamically
 create new terms in the Tag taxonomy, but any default terms can be added if
 needed.
 
+All taxonomy terms are added and edited on '/admin/structure/taxonomy'
```
Comment (rimi-itk) [inline docs/configuring-os2loop.md:31]: For consistency with other admin paths in the documentation you should enclose the path in backticks, i.e. `` `/admin/structure/taxonomy` `` (cf. https://en.wikipedia.org/wiki/Backtick).

File: docs/configuring-os2loop.md [hunk 2]
```diff
@@ -24,10 +24,20 @@ In order to make the media library work, go to
...
+
+Category: Is used for overall grouping of content e.g. "Vejledning" or "Instruks".
+Subject: Is used for grouping content related to similar themes (subjects)
+e.g. "Support" or "Administration".
+Tags: Is used as inspirational words for similar content
+e.g. "Ernæringsindsats" or "Hjælpemidler".
+As opposed to the other taxonomy terms these can also be defined by
+end users when submitting questions and/or answers.
```
Comment (rimi-itk) [inline docs/configuring-os2loop.md:39]: The list of taxonomies will be shown as a single paragraph (cf. https://github.com/os2loop/os2loop/blob/949194075f85fcdf56005968227cc9dacd24d9d5/docs/configuring-os2loop.md#taxonomies). Consider using a list to make it easier to read (https://www.markdownguide.org/basic-syntax/#lists-1).

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Some minor stylistic changes requested.

File: docs/configuring-os2loop.md (hunk 2 above)
Comment (Anna-itk) [reply docs/configuring-os2loop.md]: Made a list of taxonomy terms

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**